File: src/main/java/net/softbell/bsh/dto/request/NodeGroupDto.java
package net.softbell.bsh.dto.request;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

/**
 * @Author : Bell(bell@softbell.net)
 * @Description : 그룹 노드 정보 DTO
 */
@Getter
@Setter
@ToString
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class NodeGroupDto
{
    private boolean enableStatus;
    private String name;
    private List<Long> nodeId;
}

File: src/main/kotlin/net/softbell/bsh/dto/request/NodeGroupDto.kt
package net.softbell.bsh.dto.request

/**
 * @author : Bell(bell@softbell.net)
 * @description : 그룹 노드 정보 DTO
 */
data class NodeGroupDto (
        var enableStatus: Boolean,
        var name: String,
        var nodeId: List<Long>
)